[SYCL][Graph] Add graph query function ... (#351)
reble/llvm: sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

immediately for execution, `queue_state::executing` is returned. Otherwise,
`queue_state::recording` is returned where commands are redirected to a `command_graph`
object.
|
[source,c++]
----
command_graph<graph_state::modifiable>

> queue::ext_oneapi_get_graph();

Author: ```suggestion
queue::ext_oneapi_get_graph() const;
```

----

| Query the underlying command graph of a queue when recording.

> Returns: A copy of the graph object, the queue is recording commands into.

Reviewer: ```suggestion
Returns: The graph object that the queue is recording commands into.
```
I'd remove the word "copy" from this sentence. The command_graph objects are reference counted so we don't need this language, and "copy" might make users think that if they modify the returned command_graph object in some way, e.g. add a node, that wouldn't affect the original.
